src/components/Sidebar/CalendarPanel.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <SidebarPanel :isInSidebar="isInSidebar" header="calendar" headerIcon="@/assets/calendar.svg">
-    <vc-date-picker is-range v-model="dates" color="yellow" is-dark is-inline />
+  <SidebarPanel v-if="notableDates" :isInSidebar="isInSidebar" header="calendar" headerIcon="@/assets/calendar.svg">
```

Comment:
Instead of using v-if on this, could we just not set any date highlights if they're not provided?

Reply:
This is already happening, if there aren't any dates provided then nothing on the month is highlighted. the v-if is preventing errors from coming up when going to the actual news pages, saying that "notableDates" is undefined.

Reply:
This would hide the entire panel though, yes? My suggestion is to change the next line to something like `:attributes="notableDates?.selectAttribute || []"`

Reply:
Would it be better to put a getter that refers to `this.notableDates? this.notableDates.selectAttribute: []`?  It would make it a lot more readable imo

Reply:
Up to you